include/z64scene.h

```diff
@@ -56,7 +56,7 @@ typedef struct {
 typedef struct {
     /* 0x00 */ u8  code;
     /* 0x01 */ u8  data1;
-    /* 0x02 */ char pad[2];
+    /* 0x02 */ s8 pad[2];
```

Comment:
We use `s32` for stack padding, so I used `s8` here. iirc in ido `char` is `u8` (?) but it doesn't matter

Personally I think using signed types makes more sense because that's the default in C (without `unsigned` qualifiers)

And if `char` is guaranteed to be 1 byte then I'm okay with reverting to char

Reply:
Yes, standard says always one byte: https://en.cppreference.com/w/c/language/arithmetic_types (would kind of have to be, since a byte is the smallest addressable unit). And `char` arrays are used for struct padding universally in this codebase (and most other N64 codebases), so it seems reasonable to stick to it.